shared/triage_metadata.go

```diff
@@ -265,6 +265,23 @@ func (tm triageMetadata) Triage(metadata MetadataResults) error {
 	}
 
 	triagedMetadataMap := addToFiles(metadata, filesMap, tm.logger)
-	tm.metadataGithub.wptmetadataGitHubInfo = getWptmetadataGitHubInfo(tm.ctx, tm.githubClient)
+	tm.MetadataGithub.wptmetadataGitHubInfo = getWptmetadataGitHubInfo(tm.ctx, tm.githubClient)
 	return tm.createWPTMetadataPR(triagedMetadataMap)
 }
+
+// GetTriageMetadata returns an instance of triageMetadata struct.
+func GetTriageMetadata(ctx context.Context, git MetadataGithub, logger Logger, httpClient *http.Client) TriageMetadataInterface {
+	return triageMetadata{
+		ctx:            ctx,
+		MetadataGithub: git,
+		logger:         logger,
+		httpClient:     httpClient}
+}
+
+// GetMetadataGithub returns an instance of MetadataGithub struct.
```

Comment:
```suggestion
// GetMetadataGithub returns an instance of the MetadataGithub struct.
```

(Again, nit, the comment would be more useful if it explained what the MetadataGithub struct was for)